atlasdb-config/src/main/java/com/palantir/atlasdb/config/AtlasDbRuntimeConfig.java

```diff
@@ -77,4 +77,8 @@ public QosClientConfig qos() {
     public static ImmutableAtlasDbRuntimeConfig defaultRuntimeConfig() {
         return ImmutableAtlasDbRuntimeConfig.builder().build();
     }
+
+    public static ImmutableAtlasDbRuntimeConfig withoutSweep() {
```

Comment:
Can we make this `withSweepDisabled`. Or maybe `withDisabledSweep`?

Reply:
Sure